### DISCUSSION TITLE
Plugin and project name change

### DISCUSSION THREAD

**Discussion by Mara-Li** [ObsidianPublisher/obsidian-github-publisher#349] [Announcements]
Hello!
I renamed the plugin "Enveloppe" to clarify the difference between me and the obsidian team (and also the difference between Publisher and Publish).

All links should be already updated (including in the manifest). 

The purpose of the plugin doesn't change, btw.

**Comment by zaiquiriw** [comment]
I like it! Hopefully It's a bit more distinguished from the long list of plugins that come up when I google "publish".

**Comment by ooker777** [comment]
I'm interested to know why you choose that name. Can you share? :P

**Reply by Mara-Li** [reply]
Since day one, I always imagined the plugin like an Enveloppe. It's difficult to explain but when I tried and searched to create an icon, the image of the Enveloppe always go into my mind, without understanding why.

After discussion on the discord, I realized that the plugin have litteraly the purpose of an enveloppe, helping sending files from a place into another. 

Also I think the original idea for the icon come with some mail icon that use an enveloppe to symbolize email. 

Also the sound of enveloppe is short, and memorable. It's some self explanatory. 

That's why I choose that!

**Reply by ooker777** [reply]
Your mind is interesting. In my mind sending files from a place to another seems to describe the essence of git, not really the plugin. So the the idea of an enveloppe as an icon for the plugin is kinda out of my imagination. I wonder if the autism has some effect on this?

Nevertheless, I still glad that you found the perfect name for it. In my team we used to joke that as our project vault is kinda the source of truth, it must be the God. Therefore Git must be the angel sending our wishes to the God. We can now extend the joke that the enveloppe is what we give the angel. That's so funny 😂😂😂
